### ISSUE TITLE
11 years later

### ISSUE THREAD

**Issue by wheelq** [nlfiedler/timedog#16]
11 years later...and it still works 👯

**Comment by nlfiedler** [comment]
Perl is amazing.

**Comment by davidfstr** [comment]
Yes, thank you @nlfiedler for this project. Was able to debug why my incremental TM backups were so large recently with timedog.

(FWIW, I'm still hanging on to macOS 10.14 Mojave for a little bit longer before upgrading to macOS 11.)
